src/Symfony/Component/Mailer/Event/FailedMessageEvent.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Event;
+
+use Symfony\Component\Mime\RawMessage;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class FailedMessageEvent extends Event
+{
+    public function __construct(
+        private RawMessage $message,
+        private \Throwable $error,
```

Comment:
Why not public and no getter ?

For further potential BC handling?

Reply:
I don't get it, there is a `getError()` method here :confused: